app/components/referrals-page/referral-feature-cards.ts

```diff
@@ -12,9 +12,10 @@ interface Signature {
 }
 
 type Feature = {
-  title: string;
   bodyMarkdown: string;
   imageUrl: string;
+  popover?: { targetId: string; html: string };
```

Comment:
Let me try again. I was stumped by how to pass in the EmberPopover component using the `featureList`.

Reply:
👍 Ah, yep - might not be possible to use `FeatureList` for the content here, so that stuff might need to be in the template. Something similar: 

https://github.com/codecrafters-io/frontend/blob/7c5c8e50d5207d5280a3440a463e51ab6f142961/app/components/course-page/course-stage-step/first-stage-tutorial-card.hbs#L10-L26

Everything except a step's "body" are included in `Step`, but the body itself needs to be written inside the template

Reply:
@rohitpaulk Is this in the right direction?

<img width="1664" alt="image" src="https://github.com/user-attachments/assets/7fc4e759-4b29-4db0-9e91-1b51b535b944">

But I'm still having troubles eliminating `popover?` from `Feature`:

```
type Feature = {
  bodyMarkdown: string;
  imageUrl: string;
  popover?: { targetId: string; html: string };
  title: string;
};
```
Let me peruse `ExpandableStepList` example a bit more.